apps/site/src/pages/settings/billing-settings-panel/free-or-hobby-subscription-tier-overview.tsx

```diff
@@ -0,0 +1,570 @@
+              })}
+            />
+            <Typography
+              sx={{
+                textTransform: "uppercase",
+                display: { sx: "none", md: "block" },
+                color: ({ palette }) => palette.purple["40"],
+                fontWeight: 800,
+                fontSize: 14,
+              }}
+            >
+              Current Plan
+            </Typography>
+          </Box>
+        ) : null}
+        <Box
+          sx={{
+            padding: 4,
+            backgroundColor: "#FBF7FF",
+            borderBottomStyle: "solid",
+            borderBottomfontSize: 1,
+            borderBottomColor: ({ palette }) => palette.purple[20],
+            borderTopLeftRadius: 14,
+            borderTopRightRadius: {
+              xs: 14,
+              md: 0,
+            },
+          }}
+        >
+          <Box display="flex" alignItems="center">
+            <Typography sx={{ fontSize: 28 }}>
+              {/* @todo: dynamically determine the subscription price/currency @see: https://app.asana.com/0/0/1203781148500081/f */}
+              <strong>$2</strong>/month
+            </Typography>
+            <Typography sx={{ marginLeft: 3, fontSize: 14, fontWeight: 600 }}>
+              HOBBY
+            </Typography>
+          </Box>
+          <Typography
+            component="p"
+            variant="bpSmallCopy"
+            gutterBottom
+            sx={{
+              color: ({ palette }) => palette.purple[50],
+              fontWeight: 400,
+            }}
+          >
+            For casual users of blocks
+          </Typography>
+          <CustomLinkButton
+            href={{ pathname: "/upgrade", query: { tier: "hobby" } }}
+            variant="primary"
+            endIcon={
+              currentSubscriptionTier === "free" ? (
+                <ArrowRightIcon />
+              ) : undefined
+            }
+            disabled={currentSubscriptionTier === "hobby"}
+            size="small"
+          >
+            {currentSubscriptionTier === "hobby"
+              ? "Your current plan"
+              : "Continue"}
+          </CustomLinkButton>
+        </Box>
+        <Box
+          sx={({ spacing }) => ({
+            padding: spacing(2, 4),
+            backgroundColor: "#FBF7FF",
+            flexGrow: 1,
+          })}
+        >
+          <Typography
+            gutterBottom
+            component="p"
+            variant="bpSmallCopy"
+            sx={({ palette }) => ({ color: palette.purple[80] })}
+          >
+            <strong>Includes the following each month:</strong>
+          </Typography>
+          <Box component="ul">
+            {paidSubscriptions.hobby.coreFeatures.map((feature, index) => (
+              <SubscriptionFeatureListItem
+                // eslint-disable-next-line react/no-array-index-key
+                key={index}
+                feature={feature}
+              />
+            ))}
+          </Box>
```

Comment:
Since all feature lists have a header, we could abstract this into a component with the feature list array and header string (styling of the header could be done through headerProps component). What do you think?

Reply:
Sure, I've added a `SubscriptionFeatureList` component in https://github.com/blockprotocol/blockprotocol/pull/900/commits/0eda0ce9611cf5153d918828530d5c314ade0f20

Reply:
Sure, I've added a `SubscriptionFeatureList` component in https://github.com/blockprotocol/blockprotocol/pull/900/commits/0eda0ce9611cf5153d918828530d5c314ade0f20